docs/governance/overview.md

```diff
@@ -19,54 +19,94 @@ This page provides an overview of the On-chain Governance module that will be av
+After a proposal is created, the following rules will apply: 
+- If the proposal passes or fails normally, the fee is refunded to the issuer.  
+- If the proposal is vetoed, the fee is slashed (transferred to the **Community Governance Pool**) or reduced by the configured `LostProposalFee`.  
+- Proposals cannot be duplicated: the same commit hash cannot be submitted twice.
+- Proposals can be **cancelled** by the issuer **before the voting period starts**. (introduced in v1.10.0 Barnard release)
+- Lost proposal fees may either be accumulated in the governance contract (retrievable by the contract owner) or redirected to a **Community Governance Pool**, depending on configuration.  
 
-The quorum is computed as the sum of the staked EGLD for all addresses that cast votes.
 
-The votes will be added for each category (**Yes**, **No**, **Abstain** and **Veto**). The vote is computed as `vote = total_staked_value` for each address that cast a vote.
+### Voting
+- The voting starts from the provided starting epoch and lasts exactly *10 days*.
+- There are four vote types: **Yes**, **No**, **Abstain**, and **Veto**.  
+- Voting consumes gas (approx. 6 million units).  
+- Voting power is derived from staked and delegated EGLD.  
+- Delegation and liquid staking () contracts can cast votes on behalf of users.
+- Voting power is **linear** with stake: the more EGLD staked or delegated, the higher the voting power.  
+- The contract tracks both **used voting power/stake** (applied in a proposal) and **total available voting power/stake** for each address.  
 
+### Quorum and thresholds
 A proposal can pass only if all conditions are met:
-- the quorum value is at least the minimumQuorumThresholdPercentage * total staked value held by the staking contracts;
-- the **Yes** value > **No** value (simple majority);
-- the **Yes** value is at least the minimumPassThresholdPercentage * sum of votes on all 4 categories.
-- the **Veto** value did not reach the minimumVetoThresholdPercentage * sum of votes on all 4 categories;
-
-[comment]: # (mx-context-auto)
 
-### Example
-Let's suppose we have the following addresses that cast the following votes:
-- `alice`: staked value 2000 EGLD that vote **Yes**
-- `bob`: staked value 3000 EGLD that vote **Yes**
-- `charlie`: staked value 4000 EGLD that vote **Yes**
-- `delta`: staked value 1500 EGLD that vote **No**
+- **Quorum**: at least `MinQuorum%` of the total voting power must participate. (currently at least **20%** of total voting power)
+- **Acceptance threshold**: YES / (YES + NO + VETO) ≥ **66.67%** (`MinPassThreshold`) 
+- **Veto threshold**:  if VETO votes exceed **33%** of total participating voting power, the proposal is rejected and the proposal fee is slashed.
 
-The quorum in this case will be a value `(2000+3000+4000+1500) * 10^18 = 10500 * 10^18`.
+### Closing and cleanup
+- Proposals can only be closed after the end epoch has passed.  
```

Comment:
Can be closed also before the voting period has started, but only by the issuer. 
Anyone can close them after the voting period ended.